drakrun/drakrun/test/test_lvm.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+
+from drakrun.config import InstallInfo
+from drakrun.storage import LvmStorageBackend
+from drakrun.util import safe_delete
+import subprocess
+import os
+import logging
+import tempfile
+import secrets
+import string
+import parted
+import time
+
+
+@pytest.fixture(scope="module")
+def setup():
+    temp_file_name = tempfile.NamedTemporaryFile(delete=False).name
+    print(temp_file_name)
+    subprocess.run(['dd', 'if=/dev/zero', f'of={temp_file_name}', 'bs=1M', 'count=100'], stderr=subprocess.STDOUT, check=True)
+    loopback_file = subprocess.check_output(['losetup', '-f', '--show', temp_file_name], stderr=subprocess.STDOUT).decode().strip('\n')
+
+    # v is being added in start to ensure the lvm volume group starts with a letter
+    # don't know if it is required or not
+    volume_group = 'v' + ''.join(secrets.choice(string.ascii_letters + string.digits) for i in range(5))
+
+    # pvcreate is automatically used by this internally
+    subprocess.check_output(['vgcreate', volume_group, loopback_file], stderr=subprocess.STDOUT)
+
+    def install_patch():
+        return InstallInfo(
+            vcpus=1,
+            memory=512,
+            storage_backend='lvm',
+            disk_size='25M',
+            iso_path=None,  # not being required
+            zfs_tank_name=None,
+            lvm_volume_group=volume_group,
+            enable_unattended=None,
+            iso_sha256=None
+        )
+    InstallInfo.load = install_patch
+    InstallInfo.try_load = install_patch
```

Comment:
This will break InstallInfo class throughout the execution.
pytest has dedicated support for monkeypatching and mocking, please use it.
Additionaly, InstallInfo could be provided as a fixture to functions that load is explicitly

Reply:
We need the new `InstallInfo` only throughout the execution, we don't need the original one. So, isn't it safe to just override this?